frontend/src/metabase/parameters/components/widgets/ParameterFieldWidget/ParameterFieldWidget.jsx

```diff
@@ -118,69 +119,78 @@ export default class ParameterFieldWidget extends Component {
       );
     } else {
       return (
-        <Popover hasArrow={false} onClose={() => focusChanged(false)}>
-          <div
-            className={cx("relative PopoverBody--marginBottom", {
-              p2: !isEqualsOp,
-            })}
-          >
-            {verboseName && !isEqualsOp && (
-              <div className="text-bold mb1">{verboseName}...</div>
-            )}
-
-            {_.times(numFields, index => {
-              const value = multi ? unsavedValue : [unsavedValue[index]];
-              const onValueChange = multi
-                ? newValues => this.setState({ value: newValues })
-                : ([value]) => {
-                    const newValues = [...unsavedValue];
-                    newValues[index] = value;
-                    this.setState({ value: newValues });
-                  };
-              return (
-                <FieldValuesWidget
-                  key={index}
-                  className={cx("input", numFields - 1 !== index && "mb1")}
-                  value={value}
-                  parameter={parameter}
-                  parameters={parameters}
-                  dashboard={dashboard}
-                  onChange={onValueChange}
-                  placeholder={placeholder}
-                  fields={fields}
-                  autoFocus={index === 0}
-                  multi={multi}
-                  disableSearch={disableSearch}
-                  formatOptions={
-                    operator && getFilterArgumentFormatOptions(operator, index)
+        <TippyPopover
+          reference={this.props.target}
```

Comment:
Since this `Popover` uses its parent as an implicit target. https://github.com/metabase/metabase/blob/60502ec8acc8de8eecf07a37822437c9f807d95a/frontend/src/metabase/components/Popover/Popover.jsx#L286-L288

And `TippyPopover` always requires the trigger to be passed. In this case it uses https://github.com/atomiks/tippyjs-react#reference-reactrefobject--element